### ISSUE TITLE
Question:I know IronPython3 is in progress,will this work for ipy3?

### ISSUE THREAD

**Issue by BenDerPan** [PMoureu/iph#1]
This is a great work, and I know  [IronPython3](https://github.com/IronLanguages/ironpython3) is in progress,will this work for ipy3?

**Comment by PMoureu** [comment]
Glad you like it, thank you for your feedback ! 
I've not studied this option _yet_, i need to know exactly what will change with ipy3.
I have some changes in mind for _iph_ too, to make it more flexible and allow more custom behaviors. I will try the version3 as soon as a strong release is available.

And if you have some ideas, any contribution is welcome !

**Comment by BenDerPan** [comment]
@PMoureu I am reading iph source code now , and I wish to contribute for it . I extracted  some of it without ui parts for a easy test, and  i am try to make it work for  debug (set breakpoints) ipy script in  a embed C# application. I am not sure if iph support to do this.   :)

my repo: [IPyTracer](https://github.com/BenDerPan/IPyTracer)

**Comment by PMoureu** [comment]
i'm not sure it will perform the breakpoints as you expect in this context, it depends on how threads are handled. If your script is able to import and execute iph, it should be able to collect the objects and display the attributes, etc..  i need a small example to confirm that.

**Comment by BenDerPan** [comment]
yes, in my example it can collect the objects and get the attributes, if it can run in a debug model, and break when there is a breakpoint will be perfect.  However it's a tough target.

**Comment by ximuxingchen** [comment]
thanks
